deployment/ccip/changeset/test_helpers.go

```diff
@@ -1016,3 +1014,124 @@ func deployTransferTokenOneEnd(
 
 	return tokenContract.Contract, tokenPool.Contract, nil
 }
+
+// MintAndAllow mints tokens for deployers and allow router to spend them
+func MintAndAllow(
+	t *testing.T,
+	e deployment.Environment,
+	state CCIPOnChainState,
+	owners map[uint64]*bind.TransactOpts,
+	tkMap map[uint64][]*burn_mint_erc677.BurnMintERC677,
+) {
+	for chain, tokens := range tkMap {
+		for _, token := range tokens {
+			twoCoins := new(big.Int).Mul(big.NewInt(1e18), big.NewInt(2))
+
+			owner, ok := owners[chain]
+			require.True(t, ok)
+
+			tx, err := token.Mint(
+				owner,
+				e.Chains[chain].DeployerKey.From,
+				new(big.Int).Mul(twoCoins, big.NewInt(10)),
+			)
+			require.NoError(t, err)
+			_, err = e.Chains[chain].Confirm(tx)
+			require.NoError(t, err)
+
+			tx, err = token.Approve(e.Chains[chain].DeployerKey, state.Chains[chain].Router.Address(), twoCoins)
+			require.NoError(t, err)
+			_, err = e.Chains[chain].Confirm(tx)
+			require.NoError(t, err)
+		}
+	}
+}
+
+// TransferAndWaitForSuccess sends a message from sourceChain to destChain and waits for it to be executed
+func TransferAndWaitForSuccess(
```

Comment:
I think we need a ticket to move all these funcs out of `changeset` and into a `cciptesthelpers` package or something, `changeset` is supposed to be the public API and shouldn't have these kinda funcs.